doc/isCurrent.md

```diff
@@ -0,0 +1,67 @@
+@function can-route.isCurrent isCurrent
+@parent can-route.static
+
+@description Check if data represents the current route.
+
+@signature `route.isCurrent(data [,subsetMatch] )`
+
+  Compares `data` to the current route. Used to verify if an object is
+  representative of the current route.
+
+  ```js
+  import {route} from "//unpkg.com/can@5/core.mjs";
```

Comment:
import 'can'


(make sure to check other examples)

Reply:
There was one in deparam.md as well.